test/test_tcp.rs

```diff
@@ -578,8 +578,8 @@ fn connect_error() {
     let poll = Poll::new().unwrap();
     let mut events = Events::with_capacity(16);
 
-    // Pick a "random" port that shouldn't be in use.
-    let l = TcpStream::connect(&"127.0.0.1:38381".parse().unwrap()).unwrap();
+    let l = TcpStream::connect(&"10.245.111.123:38381".parse().unwrap()).unwrap();
```

Comment:
This could be a valid address in the environment where the tests are run.  If you want an address that will never be in use, you should use an RFC5737 address, reserved for documentation, like 192.0.2.1.
https://tools.ietf.org/html/rfc5737

Reply:
ah, good thought.